README.md

```diff
@@ -63,8 +69,46 @@ or you can use the shortcut:
 yarn fe-container test
 ```
 
+### Run tests e2e
+
+To run the e2e tests you should first of all run inside the `fe-container` directory the command
+
+```shell
+docker build -t miaplatform/microlc .
```

Comment:
we generally spell `mia-platform` rather than `miaplatform` is there any constraint for the dash to be removed?

Reply:
`miaplatform` was set at the beginning by @davidebianchi in the GitHub workflow and we kept it. Do we change it?